CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # App Center SDK for React Native Change Log
 
+## Version 2.0.0
+
+Version 2.0.0 has a **breaking change**, it only supports Xcode 10.0.0+.
+
+### AppCenter Crashes
+
+#### iOS
+
+* **[Fix]** Print an error and return immediately when calling [MSCrashes notifyWithUserConfirmation:] with confirmation handlers not implemented.
+
+### AppCenter Push
+
+#### iOS
+
+* **[Fix]** Fix updating push installation when setting the user identifier via [MSAppCenter setUserId:].
```

Comment:
This is javascript SDK.

Reply:
okay will remove this section then

Reply:
No I mean in Javascrip its `AppCenter.setUserId`.

Reply:
oh my mistake, I will edit that to be `AppCenter.setUserId`